### PR TITLE
New unit tests for maps functionality

### LINKED ISSUE #102: Adding extra unit tests to Location Info Api
Extra unit test will be added to location info api at maps branch.

### PR DESCRIPTION
**Pull request by umutoksuz**
I have aded various new unit tests for location api. And it is ready to merge with master.

### CODE REVIEW

File: practice_app/pages/index.js [hunk 1]
```diff
@@ -86,33 +84,36 @@ export default function Home({ context }) {
     const onConvertButtonClick = async () => {
         try {
             const ipdata = await getUsersIP()
-            const pricedata = await priceConverter(10, ipdata.query, "USD")
+            const pricedata = await priceConverter(10, ipdata.query, 'USD')
             Promise
```
Comment (MehdiSaffar) [inline practice_app/pages/index.js]: What is this `Promise` line by itself?

File: practice_app/services/getLocationInfo.spec.js [hunk 2]
```diff
@@ -1,4 +1,174 @@
 const { processLocationInfo } = require('./getLocationInfo')
+const { getLanguage } = require('./getLocationInfo')
+const { getCurrency } = require('./getLocationInfo')
+const { getCountry } = require('./getLocationInfo')
```
Comment (MehdiSaffar) [inline practice_app/services/getLocationInfo.spec.js]: Import all functions in the same line with `const { processLocationInfo, getLanguage, getCountry, getCurrency } = require('./getLocationInfo')`

File: practice_app/services/getLocationInfo.spec.js [hunk 3]
```diff
@@ -1,4 +1,174 @@
...
+        const input = {
+            country: 'Turkey',
+        }
+
+        const output = getLanguage(input)
+
+        expect(output).toBe(undefined)
+    })
+
+    test('returns nothin when country_module.languages doesnt exist.', () => {
```
Comment (MehdiSaffar) [inline practice_app/services/getLocationInfo.spec.js]: nothin typo

File: practice_app/services/getLocationInfo.spec.js [hunk 4]
```diff
@@ -1,4 +1,174 @@
...
+                        code: 'b',
+                        name: 'c',
+                    },
+                ],
+            },
+        }
+
+        const output = getLanguage(input)
+
+        expect(output).toBe(undefined)
```
Comment (MehdiSaffar) [inline practice_app/services/getLocationInfo.spec.js]: Sometimes you use .toBe(undefined) other times .toEqual(undefined). For the sake of consistency please use .toStrictEqual(undefined) everywhere

File: practice_app/services/getLocationInfo.spec.js [hunk 5]
```diff
@@ -1,4 +1,174 @@
...
+    })
+
+    test('returns nothin when country_module.languages has 0 elements', () => {
```
Comment (MehdiSaffar) [inline practice_app/services/getLocationInfo.spec.js:149]: nothin typo

**Review by MehdiSaffar — CHANGES_REQUESTED** [review]

File: practice_app/services/getLocationInfo.spec.js (hunk 2 above)
Comment (umutoksuz) [reply practice_app/services/getLocationInfo.spec.js]: I have fixed this

File: practice_app/services/getLocationInfo.spec.js (hunk 3 above)
Comment (umutoksuz) [reply practice_app/services/getLocationInfo.spec.js]: fixed

File: practice_app/services/getLocationInfo.spec.js (hunk 4 above)
Comment (umutoksuz) [reply practice_app/services/getLocationInfo.spec.js]: fixed

File: practice_app/services/getLocationInfo.spec.js (hunk 5 above)
Comment (umutoksuz) [reply practice_app/services/getLocationInfo.spec.js:149]: fixed

**Review by MehdiSaffar — APPROVED** [review]
Good job, can merge

**Comment by omerfdeniz** [comment]
Good job, clean code!

**Pull request merged**